packages/jest-runner/fuzz.ts

```diff
@@ -95,6 +95,7 @@ export const runInFuzzingMode = (
 	fuzzerOptions: string[]
 ) => {
 	fuzzerOptions.unshift(corpus.inputsDirectory);
+	fuzzerOptions.unshift(corpus.corpusDirectory);
```

Comment:
Could you add a comment to emphasize that the `corpus.corpusDirectory` must be the first input directory provided to libFuzzer so that generated inputs are created there?